apps/chat/tasks.py

```diff
@@ -26,6 +27,36 @@ def periodic_tasks(self):
         _check_future_messages()
 
 
+@shared_task
+def send_bot_message_to_users(*args, **kwargs):
```

Comment:
why use `*args, **kwargs` and not normal arguments?

Reply:
Initially when I tested it it didn't work, so I just used args + kwargs and forgot about it tbh. I checked again and it worked now. I think I misspelled a parameter previously. Thanks for raising this